nucypher/network/server.py

```diff
@@ -252,6 +258,11 @@ def reencrypt():
                     error = (message, HTTPStatus.FORBIDDEN)
                     log.info(message)
                     return Response(message, status=error[1])
+                except ReencryptionCondition.ConditionEvaluationFailed as e:
+                    message = f"Decryption condition not evaluated: {e}"
```

Comment:
In this block the condition was evaluated but the result was false.  It's more accurate to say something along the lines of "decryption conditions unfulfilled"

Reply:
This is not the actual condition failing (i.e. there is no condition result), but rather the failure of the function that is needed to determine the value to use to check the condition.

Reply:
Ah, okay I misunderstood.

Reply:
In that case, perhaps the name of the exception can be made clearer? I was trying not to use `Function` in the name so that `ReencryptionCondition` can remain sufficiently abstract. Maybe something like `ConditionValueNotObtained` or `ConditionCheckNotEvaluated`...?

Reply:
Is it a matter of validity?